repro/ResponseContext.cxx

```diff
@@ -1533,8 +1533,8 @@ ResponseContext::forwardBestResponse()
    
    if(mBestResponse.header(h_StatusLine).statusCode() == 503)
    {
-      //See RFC 3261 sec 16.7, page 110, paragraph 2
-      mBestResponse.header(h_StatusLine).statusCode() = 480;
+      // See RFC 3261 sec 16.7, page 111, paragraph 2
+      mBestResponse.header(h_StatusLine).statusCode() = 500;
```

Comment:
FYI - I have reached out to @docfaraday to see if he can remember back 18 years ago, as to why this change was made in the first place. :)

Reply:
I believe my solution should work without changes to follow RFC and with some extra changes with a derived class if you want some custom SIP code here.

Reply:
This was docfaraday's response:
     I'm not really sure. However, 500 makes almost no sense to me. It
gives no hint at all what went wrong, or where (did it fail at the
proxy? downstream?). That might be why I changed it? 480 at least would
suggest an appropriate course of action for the human caller (ie; try
again later), but it has its own problems too though. I don't know that
there actually is a good option.

I think we are good to proceed with the change to follow the RFC, and if someone wants to override then they can.